.github/workflows/validator-update-pr.yml

```diff
@@ -37,56 +37,44 @@ jobs:
         with:
           data: |
             {
-              "validatorVersion": "${{ github.event.inputs.release_version }}"
+              "validatorVersion": "${{ github.event.inputs.validator_version }}"
             }
 
-      - run: echo "validator version - ${{ steps.info.outputs.version }} - ${{ github.event.inputs.release_version }}"
-
-      - name: Install 🔧
-        run: npm install
-
-      - name: Build 🔧
-        run: npm run build
+      - run: echo "validator version - ${{ steps.info.outputs.version }} - ${{ github.event.inputs.validator_version }}"
 
       - name: Commit files from the build
         run: |
           git config --local user.name  ${{ github.actor }}
           npm run prettier:fix
           git add --all
-          git commit -m "Release PR commit for ${{ github.event.inputs.release_version }}"
+          git commit -m "Validator version update for ${{ github.event.inputs.validator_version }}"
 
       - name: Push changes
         uses: ad-m/github-push-action@master
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          branch: release-pr-${{ github.event.inputs.release_version }}
-
-      - name: Tag release
-        uses: rickstaa/action-create-tag@v1
-        with:
-          tag: ${{ github.event.inputs.release_version }}
-          message: "Validator Release ${{ github.event.inputs.release_version }}"
```

Comment:
We don't want to create a tag here since the publish.yml workflow does that now.